src/richchk/io/richchk/lookups/swnm/rich_swnm_rebuilder.py

```diff
@@ -21,7 +23,9 @@ class RichSwnmRebuilder:
     _LOG = logger.get_logger("RichSwnmRebuilder")
 
     @classmethod
-    def rebuild_rich_swnm_from_rich_chk(cls, rich_chk: RichChk) -> RichSwnmSection:
+    def rebuild_rich_swnm_from_rich_chk(
+        cls, rich_chk: RichChk
+    ) -> Tuple[RichSwnmSection, RichSwnmLookup]:
```

Comment:
sadly this step is required so we can resolve switches without indices to their index.  

Another approach could be to modify the data in place but I would prefer to reduce or eliminate data mutations/side effects as much as possible.